### ISSUE TITLE
ENHANCEMENT: Change Password / Edit Profile URL filters

### ISSUE THREAD

**Issue by andrewlimaza** [strangerstudios/paid-memberships-pro#1196]
**Is your feature request related to a problem? Please describe.**
We should make the account page Change Password and Edit Profile links filterable.

https://github.com/strangerstudios/paid-memberships-pro/blob/dev/shortcodes/pmpro_account.php#L115-L116

**Comment by philwp** [comment]
I agree.  This would be very helpful.  I need to use a front-end password reset and had to hide the pmpro_actionlink div.  

I created my own links and hooked that to `mpro_account_bullets_bottom`  which is not optimal

**Comment by dparker1005** [comment]
Linking to https://github.com/strangerstudios/paid-memberships-pro/issues/1148

**Comment by dparker1005** [comment]
Closing this as completed. The account page profile links are now built through the `pmpro_account_profile_action_links` filter, so custom code can replace the Edit Profile and Change Password links without hiding and rebuilding the whole action area.

The current filter name was fixed in PMPro 2.3.2.
